src/search.coffee

```diff
@@ -133,15 +114,12 @@ class Search
         ###
         data =
           estimatedCount: _.last(responses).estimatedCount
-          results: []
-
-        # Avoid duplicates by checking result IDs.
-        existingIds = {}
-        responses.forEach (response) ->
-          response.results.forEach (result) ->
-            unless result.id of existingIds
-              data.results.push result
-              existingIds[result.id] = true
+          results: _.chain responses
+            .pluck 'results'
+            .flatten()
+            .uniq false, ({url, contentUrl}) ->
+              url or contentUrl
```

Comment:
`contentUrl` is used by video/news.

Reply:
Why not leave that as a code comment?

Reply:
Wilco; didn't think of it until skimming the PR :slightly_smiling_face: